src/site/xdoc/manual/appenders.xml

```diff
@@ -1735,6 +1735,11 @@ public class JpaLogEntity extends AbstractLogEventWrapperEntity {
               <td>The Kafka topic to use. Required.</td>
             </tr>
             <tr>
+              <td>key</td>
+              <td>String</td>
+              <td>The key that will be sent to Kafka with every message.</td>
+            </tr>
+            <tr>
```

Comment:
We should add a comment on the `properties` section to not specify `key.serializer` (as well as not specify `value.serializer`).